lxdock/conf/config.py

```diff
@@ -115,6 +153,27 @@ def _get_container_config_dict(self, container_dict):
         del container_config['containers']
         return container_config
 
+    def _get_dotenv_items(self, env_filepath):
```

Comment:
I understand your point. That said other Python tools (eg. docker-compose) seem to use similar techniques to process ``.env`` files, which seems fine IMO. If we were to ``source`` these ``.env`` files, I guess this would mean we would have to do something like ``source .env && env`` and then parse the output of this command. But this would also involve declaring vars using `export` in the `.env` file if I'm correct.